highfidelity/hifi
# The client should render "something" for an avatar even if mesh or identity is not available

This pr implements an overlay that is created while an Avatar is discovered by the AvatarManager but the geometry is not loaded.  Alternatively it will display while the model is changing for an Avatar that is already in the domain.

The associated FogBugz is 16105.

For this pr the class OtherAvatar was moved from avatars-renderer to interface.  This allows the OtherAvatar class to access the application list of overlays.

### Test Plan
1. Start interface and go to a domain with other avatars in it, preferably 'rust-experiments'.  While the other avatar models are loading you should see a purple orb overlay in the place of the loading avatars.
2. Go to a domain with two different clients.  On one client, change the existing path of the avatar by typing the letter 'x' at the end of the current url.  When you click save, watch the other client to see that there is an orb for a moment while the avatar is searching the bad url and changing to default avatar.
